docs/man_pages/general/autocomplete-disable.md

```diff
@@ -20,4 +20,7 @@ Command | Description
 [error-reporting](error-reporting.html) | Configures anonymous error reporting for the NativeScript CLI.
 [help](help.html) | Lists the available commands or shows information about the selected command.
 [doctor](doctor.html) | Checks your system for configuration problems which might prevent the NativeScript CLI from working properly.
+[proxy](proxy.html) | Displays proxy settings.
+[proxy clear](proxy-clear.html) | Clears proxy settings.
+[proxy set](proxy-set.html) | Sets proxy settings.
```

Comment:
How do I update a proxy to be enabled/disabled? Can I do it via command or by manually editing the config.json?